Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.            if wis.domain_name:
</pre><p>If you change to </p><pre class="ql-syntax" spellcheck="false">if not wis.domain_name:
    continue
</pre><p> </p><p>Will let you unindent the remainder of the loop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>